README.md

```diff
@@ -63,26 +63,13 @@ Once you initialize a `Client`, you can call methods to make requests to the Okt
 
 ```csharp
 var client = new IdxClient(new IdxConfiguration()
-            {
-                Issuer = "{YOUR_ISSUER}", // e.g. https://foo.okta.com/oauth2/default, https://foo.okta.com/oauth2/ausar5vgt5TSDsfcJ0h7
-                ClientId = "{YOUR_CLIENT_ID}",
-                ClientSecret = "{YOUR_CLIENT_SECRET}", //Required for confidential clients. 
-                DeviceToken = "{YOUR_DEVICE_TOKEN}", // Client device token. Optional property. Only used for confidential clients with ClientSecret. 
```

Comment:
Did you mean to remove these sections?

Reply:
No I didn't mean to remove the Custom Headers section or the Device Token line. 
Not sure how it happened. Have it added it back in.